### PR TITLE
chore: recommend ccc

### LINKED ISSUE #739: Lead users to ckb-ecofund/ccc
**Is your feature request related to a problem? Please describe.**

CCC(https://github.com/ckb-ecofund/ccc) is the latest and recommended JavaScript SDK in the CKB ecosystem. Therefore, it's important to prominently remind users to choose CCC as their primary development tool.

**Describe the solution you'd like**

Add an annotation in the readme for this and highlight a link to CCC

### PR DESCRIPTION
**Pull request by homura**
Resolve #739

### CODE REVIEW

**Comment by Keith-CY** [comment]
Is this ready to merge?

**Comment by homura** [comment]
> Is this ready to merge?

yes, waiting for approval

**Review by Keith-CY — APPROVED** [review]

**Pull request merged**
